tools/stress-cluster/services/Stress.Watcher/tests/ResourceTest.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.Collections.Generic;
+using Xunit;
+using FluentAssertions;
+using Stress.Watcher.Extensions;
+using k8s;
+using k8s.Models;
+
+namespace Stress.Watcher.Tests
+{
+    public class ResourceTest
+    {
+        [Fact]
+        public void GetInstanceTest()
+        {
+            var schedule = new GenericChaosResource();
+            var networkChaos = new ChaosResourceSpec();
+            schedule.Spec = new ChaosResourceSpec();
+            networkChaos.Selector = new ChaosSelector() {
+                LabelSelectors = new ChaosLabelSelectors() {
+                    TestInstance = "ResourceTestTestInstance"
+                }
+            };
+            schedule.Spec.NetworkChaos = networkChaos;
+            networkChaos.GetTestInstance().Should().Be("ResourceTestTestInstance");
+            schedule.Spec?.GetTestInstance().Should().Be("ResourceTestTestInstance");
```

Comment:
It would be good to add tests for empty schedule (no inner chaos spec) and missing label as well (no TestInstance label).

Reply:
also, do schedules support multiple inner chaos specs?

Reply:
Correct me if I'm wrong, since I'm not well versed in go. The [source code](https://github.com/chaos-mesh/chaos-mesh/blob/master/api/v1alpha1/schedule_types.go#L74) seems to suggest that there can only be one type of chaos being scheduled.